packages/material-ui-codemod/README.md

```diff
@@ -11,10 +11,11 @@ APIs.
 
 ## Setup & Run
 
-- `npm install -D @material-ui/codemod`
-- `npx jscodeshift -t <codemod-script> <path>`
-- Use the `-d` option for a dry-run and use `-p` to print the output
-  for comparison
+- `npm install -D @material-ui/codemod@next`
```

Comment:
```suggestion
<!-- #default-branch-switch -->
- `npm install -D @material-ui/codemod@next`
```

So that we know when to change it.

Reply:
Ah yes, smart. I need to get this habit!